package.json

```diff
@@ -257,6 +257,7 @@
     "@fontsource/rubik": "^4.5.6",
     "@mui/icons-material": "^5.5.1",
     "@mui/material": "^5.5.1",
+    "@octokit/rest": "^19.0.4",
```

Comment:
Don't need this anymore, nextjs server does the actual github call

Reply:
Good catch, resolvec